ui/pages/confirmations/confirm-transaction-base/confirm-transaction-base.component.js

```diff
@@ -447,6 +461,14 @@ export default class ConfirmTransactionBase extends Component {
               type={SECONDARY}
               key="total-detail-text"
               value={value}
+              suffixProps={{
```

Comment:
hiya @matthewwalsh0, thanks for the reviews! I don't think it's worth doing this since there is still a lack of consistency between their instances, and we are planning to deprecate this UI with the redesign work.

These are the variations I've gathered separated by empty lines:

```
color: balanceColor, // errorDefault or textAlternative
variant: TextVariant.bodySm,

color: TextColor.textAlternative,
variant: TextVariant.bodyMd,

color: TextColor.textAlternative,
variant: TextVariant.bodySm,

color: TextColor.textAlternative,
variant: TextVariant.bodySmBold,

color: TextColor.textDefault,
variant: TextVariant.bodyMdBold,

variant: TextVariant.bodyMdBold,

font: FontWeight.Medium,
variant: TextVariant.bodyLgMedium,
```

This would include out-of-scope work from the gas UI. 

--- 

I think what we could do as a quick improvement is combine the textProps and suffixProps because they hold the same values in each instance. The new prop could be named textAndSuffixProps. I will create a follow-up PR for this work. Let me know if there are additional thoughts on this

Reply:
Sorry for the confusion, I'm not suggesting anything be done now, or even for this component specifically.

I just wanted to verify that we want to try and encapsulate styling within the components so it's not the job of the caller to provide the same styling values repeatedly.

Reply:
🙏🏼  absolutely agree. I would say it's a bit tricky abstracting and encapsulating these styles. This would help applying consistency